robot_smach_states/src/robot_smach_states/manipulation/place.py

```diff
@@ -216,12 +216,22 @@ def __init__(self, robot, item_to_place, place_pose, arm, place_volume=None, upd
         # Check types or designator resolve types
         assert(item_to_place.resolve_type == Entity or type(item_to_place) == Entity)
         assert(arm.resolve_type == PublicArm or type(arm) == PublicArm)
+        #assert(place_volume.resolve_type == str or (type(place_volume) == str))
+
+        # parse place volume
+        if place_volume is not None:
```

Comment:
1: this should not happen in the init function.
2: there is a function somewhere that does this for you.

Reply:
Should indeed be resolved 'at runtime'. 
The place_area however gets passed to an EmptySpotDesignator which currently needs a concrete value for `area`. It would be nicers if it could accept that as a designator as well. 
For the sake of speed and the scope of this PR, I'll leave that for when we really need that bit of functionality.